deployment/morph/machines/marlowe-dash.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, config, lib, ... }:
+{
+  imports = [
+    ../../../nix/modules/pab.nix
+  ];
+
+  networking = {
+    firewall.allowedTCPPorts = [ 22 80 9080 ];
+  };
+
+  services.pab = {
+    enable = true;
+    pab-package = pkgs.plutus-pab.pab-exes.plutus-pab;
+    contracts = [ "${pkgs.marlowe-app}/bin/marlowe-app" ];
+    staticContent = pkgs.marlowe-dashboard.client;
+    dbFile = "/var/lib/pab/pab-core.db";
+    defaultWallet = 1;
+    webserverPort = 9080;
+    walletPort = 8081;
+    nodePort = 8082;
+    chainIndexPort = 8083;
+    signingProcessPort = 8084;
+    metadataPort = 8085;
```

Comment:
I suspect this may be 9080, 9081, ..., because of this PR https://github.com/input-output-hk/plutus/pull/2884/files
But I don't know why they changed it, just pointing it out

Reply:
I see, well they are specifying different default ports but that doesn't really have any impact here -  As long as we are consistent with ports we are using within our deployment configuration we are good.

Good to know though, i wasn't aware of this change. Thanks @palas